.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
What about https://stackoverflow.com/questions/7335420/global-git-ignore?

Reply:
Will use that for my `go.work` files :) Many thanks!